README.md

```diff
@@ -1,2 +1,102 @@
-# helm-charts
-Iterative helm charts
+## Studio Helm Chart
+
+## Install Studio
+
+Configure Credentials to pull images from secure registry
+
+```bash
+$ kubectl create secret docker-registry iterativeai \
+    --docker-server=docker.iterative.ai \
+    --docker-username=username \
+    --docker-password=password
+```
+
+Create a file `override.yaml` with the following content or edit `values.yaml`
+
+```yaml
+imagePullSecrets:
+  - name: iterativeai
+```
+
+ Deploy Studio
+
+```bash
+$ helm dependency update
+$ helm install studio studio/ -n <namespace> -f override.yaml
```

Comment:
Probably better to leave out `-n <namespace>` because otherwise it's not clear that `global.configurations.postgresDatabaseUrl` must be updated as well